File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    answer = 0
    p_len = len(p)
    t_len = len(t)
    p = int(p)
    for i in range(0,t_len+1-p_len):
        if int(t[i:i+p_len]) <= p:
            answer+=1
    return answer

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.java
class Solution {
    public int solution(String t, String p) {
        int len = p.length();
        long pNum = Long.valueOf(p);
        int answer = 0;
        for (int i = 0; i <= t.length() - p.length(); i++){
            long tmp = Long.valueOf(t.substring(i, i + p.length()));
            if (tmp <= pNum)
                answer++;
        }
        return answer;
    }
}